cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -441,3 +457,27 @@ def test_ansible_group_id_in_payload(service_catalog, service_request, service):
     result_dict = json.loads(json_str[5].replace('", "', "").replace('\\"', '"').replace(
         '\\, "', '",').split('" ] } PLAY')[0])
     assert "group" in result_dict["manageiq"]
+
+
+@pytest.mark.provider([EC2Provider], scope="function")
+def test_embed_tower_exec_play_against_amazon(request, provider, setup_provider,
+        ansible_catalog_item, service, ansible_amazon_credential, service_catalog):
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {
+            "playbook": "list_ec2_instances.yml",
+            "cloud_type": "Amazon",
+            "cloud_credential": ansible_amazon_credential.name
+        }
+
+    def _revert():
+        with update(ansible_catalog_item):
+            ansible_catalog_item.provisioning = {
+                "playbook": "dump_all_variables.yml",
```

Comment:
I assume we're hard-coding this playbook name because it is usually the first in the drop-down? Should we revert with something a little 'smarter' that does not use a hard-coded name? I.e. revert to <whatever playbook no. 1 is>

Reply:
I don't think so, tests in this module use exactly that playbook and only few of them use other playbooks.